### ISSUE TITLE
pytest hangs after running if logger.configure called

### ISSUE THREAD

**Issue by peterschutt** [starlite-api/example-pg-docker#14]
Output after `ctrl-c`:

```python exception
^CException ignored in atexit callback: <bound method QueueListenerHandler.stop of <QueueListenerHandler (NOTSET)>>
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/starlite/logging.py", line 46, in stop
    self._listener.stop()
  File "/usr/local/lib/python3.10/logging/handlers.py", line 1559, in stop
    self._thread.join()
  File "/usr/local/lib/python3.10/threading.py", line 1089, in join
    self._wait_for_tstate_lock()
  File "/usr/local/lib/python3.10/threading.py", line 1109, in _wait_for_tstate_lock
    if lock.acquire(block, timeout):
KeyboardInterrupt: 
ERROR: 1
```

**Comment by peterschutt** [comment]
This is the reason for this.. if I un-comment, pytest hangs after running. I wonder if related to the exception that is logged after running the starlite tests, but really no idea as yet.

```python
app = Starlite(
    ...
    # enabling this causes pytest to hang
    # on_startup=[logger.configure],
    ...
)
```

**Comment by Goldziher** [comment]
lemme check this one out

**Comment by Goldziher** [comment]
issue is fixed with starlite v1.3.7

**Comment by peterschutt** [comment]
Nice thanks. Fixed the error in atexit at end of starlite tests too!

**Comment by peterschutt** [comment]
Closed in #25
